File: webapp/src/app/models/SessionRequest.ts
export interface SessionRequest {
  id: string;
  start: Date;
  end: Date;
  title: string;
  userId: string;
  status: SessionRequestStatus;
  sessionId?: string;

  color: { primary: string; secondary: string };
}

export enum SessionRequestStatus {
  Waiting,
  Scheduled,
}

File: server/Uragsha.Models/Scheduling/SessionRequest.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Uragsha.Models.Scheduling
{
    public class SessionRequest
    {
        [Key]
        public string Id { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public string Title { get; set; }
        public string UserId { get; set; }
        public SessionRequestStatus Status { get; set; }
        public string SessionId { get; set; }
    }
}
